Smile/src/main/java/smile/neighbor/LSH.java

```diff
@@ -15,10 +15,7 @@
  *******************************************************************************/
 package smile.neighbor;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
```

Comment:
It is better not using * for import. Can you please import only necessary classes?